go/mysql/mysql.go

```diff
@@ -38,6 +38,87 @@ func init() {
 }
 
 const (
+	// TypeTiny specifies a TINYINT type.
```

Comment:
It turns out that this change is not trivial because the proto3 types don't match the current field type. This ends up propagating everywhere else.
But we should be able to do this when we do the proto3 migration.